apps/src/applab/Exporter.js

```diff
@@ -715,6 +717,9 @@ export default {
     };
 
     await session.sendCodeAsync(files);
+    // TODO: avoid calling internal method on snack-sdk once the sendCodeAsync()
+    // method can be called without a debounce
+    await session._publishNotDebouncedAsync();
```

Comment:
Is there a work item associated with this TODO?

Reply:
I've brought it up with the Expo team during our last call. It is an issue with their SDK that I believe they should fix.

Reply:
Ok. In that case, can you link to an issue on their repo or perhaps even just a section of code? Just want to get some more context for the next person who reads it. i.e. how should they know that the method no longer requires a debounce.

Reply:
Sadly, they don't have an Issues section for their repo, but I updated the comment based on your suggestion.

Reply:
👍